### ISSUE TITLE
Cakphp is faster than phalcon?

### ISSUE THREAD

**Issue by johnlanz** [kenjis/php-framework-benchmark#83]
Is this normal? laravel and cakephp is faster than phalcon?
![phpperformance](https://user-images.githubusercontent.com/601899/33598704-b011ebf6-d9de-11e7-8286-5913774f3dc2.png)
I'm currently using a $20 vps from turnkeyinternet

**Comment by wmac** [comment]
Not just cakephp, but even Laravel etc. It probably is not correct. 10 million requests on a $20 VPS?

**Comment by johnlanz** [comment]
I think its 10k request which is reasonable
Although the vps is really high end for $20 you get 8  cpu E5 processor and 8GB of RAM : https://turnkeyinternet.net/cloud-hosted-virtual-servers-vps/

**Comment by cebe** [comment]
likely an error, you should check if the frameworks are actually executed and you are not benchmarking an HTTP server error page.

**Comment by johnlanz** [comment]
yeah its a server error. I really hate to say this but php frameworks are really too slow. less than 3,000 request per second. I benchmark a java web server (light 4j) on my vps and I've got 2.1 million request per second and for light 4j json API I've got 600,000 rps that's 200X faster than any php framework.

**Comment by joanhey** [comment]
The difference between a fast php framework and a java framework, are more similar when you have connections to the database. Almost all apps use database.
This benchmark output 'Hello World' for compare the frameworks minimum overhead without cache, in a context.
For output 'Hello world' is faster using php without framework or send it from the server directly.

You are comparing apples with oranges.
You said 'I benchmark a java web server .... faster than any php framework'. 
Benchmark this java web server versus apache, nginx, ...
